plugins/catalog-backend-module-bitbucket/api-report.md

```diff
@@ -0,0 +1,47 @@
+## API Report File for "@backstage/plugin-catalog-backend-module-bitbucket"
```

Comment:
I realise that we should add these modules to the no errors list.

Reply:
aighhhht, will do after so we don't have to wait for the rebuild